bcda/cclf/fileprocessor.go

```diff
@@ -119,6 +128,15 @@ func (p *processor) handleArchiveError(path string, info os.FileInfo, cause erro
 	return err
 }
 
+func isOptOut(filename string) (isOptOut bool) {
```

Comment:
Is this based on an existing function/regex? Just wondering if the CCLF and opt out importers can share the same utility function in case we ever need to change them.

Reply:
yeah - I pulled the [regex out of this file](https://github.com/CMSgov/bcda-app/blob/e83513faa7785edd9db1eefdd7c3742f1777c043/optout/utils.go#L25C6-L25C19), but can create an exported util function that both can use! I'll update.

Reply:
OK - this has now been moved to the utils.go file used by `optout` to reduce repetitive code. 🎉